test/e2e/gitops_service_test.go

```diff
@@ -237,3 +244,97 @@ func tearDownArgoCD(t *testing.T) {
 	assertNoError(t, err)
 
 }
+
+func policyRulesForClusterConfig() []rbacv1.PolicyRule {
+	return []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{
+				"operators.coreos.com",
+			},
+			Resources: []string{
+				"*",
+			},
+			Verbs: []string{
+				"*",
+			},
+		},
+		{
+			APIGroups: []string{
+				"operator.openshift.io",
+			},
+			Resources: []string{
+				"*",
+			},
+			Verbs: []string{
+				"*",
+			},
+		},
+		{
+			APIGroups: []string{
+				"user.openshift.io",
+			},
+			Resources: []string{
+				"*",
+			},
+			Verbs: []string{
+				"*",
+			},
+		},
+		{
+			APIGroups: []string{
+				"config.openshift.io",
```

Comment:
We should have a test that consumes https://github.com/openshift/openshift-gitops-examples/blob/master/argocd/builds/base/build.yaml#L2 in an Application CR

Reply:
And/or
https://github.com/openshift/openshift-gitops-examples/blob/master/argocd/image/image.yaml